contracts/bid/PartyBid.sol

```diff
@@ -100,6 +105,9 @@ contract PartyBid is Party {
         nftContract = IERC721Metadata(_nftContract);
         tokenId = _tokenId;
         auctionId = _auctionId;
+        // TODO: validate that gatedToken is a contract
```

Comment:
would it be worth calling an erc-20 specific fn call to ensure it's not just a contract but also an erc20?